pkgs/by-name/yt/yt-dlp/package.nix

```diff
@@ -17,13 +17,13 @@ python3Packages.buildPythonApplication rec {
   # The websites yt-dlp deals with are a very moving target. That means that
   # downloads break constantly. Because of that, updates should always be backported
   # to the latest stable release.
-  version = "2024.10.22";
+  version = "2024.11.4";
```

Comment:
It's not 2024.11.4, It's 2024.11.04

Reply:
also, your changelog seems to be invalid.
correct url: https://github.com/yt-dlp/yt-dlp/releases/tag/2024.11.04

Reply:
We are downloading https://pypi.org/project/yt-dlp/2024.11.4/